sklearn/utils/_repr_html/params.py

```diff
@@ -23,6 +75,31 @@ def _read_params(name, value, non_default_params):
     return {"param_type": param_type, "param_name": name, "param_value": cleaned_value}
 
 
+def _doc_row(estimator_type, row, param_name):
+    """
+    Generate an HTML table row containing a link to the online
+    documentation for a specific parameter of an estimator.
+    If the link cannot be generated, an empty string is returned.
+    """
+
+    link = link_to_param_doc(estimator_type, row)
+
+    if link:
+        link_string = (
+            f'rel="noreferrer" target="_blank" href='
+            f"{link} "
+            f'style="color: white; background: black;">?<span>Online `{param_name}` '
+            f"documentation</span>"
```

Comment:
The link to the estimator doc is written as `Documentation for ...`. Let's be consistent and keep the same structure ?

Reply:
Done